content/doc/developer/views/read-only.adoc

```diff
@@ -107,6 +113,72 @@ Removing controls from a page, like an X button on a repeatable element:
 </j:if>
 ----
 
+== Allowing access to a view
+
+Normally in Jenkins the layout tag of a page will have a permission
+attribute which governs which permission is required to access the page. 
+
+Jenkins has a concept of 'impliedBy' in it's permission, which means that each permission is allowed to be automatically granted by another one.
+_Job/ExtendedRead_ is implied by _Job/Configure_, which means that requiring _Job/ExtendedRead_
+to access a view, will also grant _Job/Configure_ access.
+
+Here are some examples that show what you will need to update the layout tag to:
+
+Requiring the _Job/ExtendedRead_ permission in Jelly:
```

Comment:
These examples should be extended to only require permissions for part of the view.

Reply:
isn't that covered here:
https://github.com/timja/jenkins.io/blob/additional-read-only-docs/content/doc/developer/views/read-only.adoc#partial-view-read-only

Reply:
Not really, those disable content, this comment is about removing it. Like Save/Apply buttons.